tests/test_interaction_group_traj.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+from jax import grad, jit
+from jax.scipy.special import logsumexp
+
+from timemachine.fe.interaction_group_traj import InteractionGroupTraj, nb_pair_fxn
+
+
+def test_ig_traj():
```

Comment:
nit: Would be good to mark this as `pytest.mark.nocuda` and to make the name of the test more explicit